### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by MChoi-git**
Added issue templates for feature requests and bugs.

### CODE REVIEW

**Review by adil-a — COMMENTED** [review]
I think it looks pretty informed and to the point. @jwilles any other suggestions here?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+title: "[BUG][...]:"
+labels: BUG
+assignees: MChoi-git
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
```
Comment (jwilles) [inline .github/ISSUE_TEMPLATE/bug_report.md:13]: These reproduction steps seem more appropriate for a web app. Can we tailor this for bugs that are likely to arise in our use case?

**Comment by jwilles** [comment]
Overall looks good to me. Minor comment on tailoring the steps to reproduce - which is important given that issues will likely be model/library/hardware dependent.

**Review by adil-a — APPROVED** [review]
I believe outstanding comments have been addressed here and we should be good to merge.

**Pull request merged**